### PR TITLE
Archives

### PR DESCRIPTION
**Pull request by sanathkumar26700**
Added Archives features like,
 
- Add to archive
- Restore from archive

### CODE REVIEW

**Review by MarkVed17 — APPROVED** [review]
Archive functionality works!
Just a suggestion to add title attribute over icons.
You are good to proceed.

File: src/Components/Navbar/Navbar.jsx [hunk 1]
```diff
@@ -68,10 +69,10 @@ function Navbar() {
                             <div className="nav-list--item__icon--wrapper">
                                 <span>
                                         <i className="fas icon fa-archive">
-                                            {isAuthorized  && <span className="status-badge notification-badge">{'0'}</span>}
+                                            {isAuthorized  && <span className="status-badge notification-badge">{archivesData.length}</span>}
                                         </i>
                                 </span>
-                                <span className="icon--text">Archives</span>
+                                <span className={`icon--text ${location.pathname === '/archives' && `active__icon--text`}`}>Archives</span>
```
Comment (msdeshmukh009) [inline src/Components/Navbar/Navbar.jsx:75]: I think you can use Navlinks here. To highlight the active link.

File: src/Utilities/JS/Data Handlers/archivesDataHandler.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+const restoreNoteFromArchivesDataHandler = async (note, token, userDataDispatch, setFetchingArchives) => {
+  setFetchingArchives(true)
+  try {
+    const response = await axios.post(
+      `/api/archives/restore/${note._id}`,
+      {  },
+      { headers: { authorization: token } }
+    );
+    console.log(response)
```
Comment (msdeshmukh009) [inline src/Utilities/JS/Data Handlers/archivesDataHandler.js:44]: You forgot to remove this console i guess

**Review by msdeshmukh009 — APPROVED** [review]
The archive feature is working fine. 
Good to go.

File: src/Components/Navbar/Navbar.jsx (hunk 1 above)
Comment (sanathkumar26700) [reply src/Components/Navbar/Navbar.jsx:75]: Thanks for the suggestion, I'll Check it out!

**Pull request merged**